mw_jwt.go

```diff
@@ -206,22 +225,41 @@ func (k *JWTMiddleware) getBasePolicyID(token *jwt.Token) (string, bool) {
 	return "", false
 }
 
+func (k *JWTMiddleware) getUserIdFromClaim(token *jwt.Token) (string, error) {
+	var userId string
+	var found bool
+
+	if k.Spec.JWTIdentityBaseField != "" {
+		userId, found = token.Claims.(jwt.MapClaims)[k.Spec.JWTIdentityBaseField].(string)
```

Comment:
we could reduce number of type asserts storing `token.Claims.(jwt.MapClaims)` in variable and use after